datumaro/components/launcher.py

```diff
@@ -46,7 +49,13 @@ def __iter__(self):
                     continue
                 inputs.append(np.atleast_3d(item.media.data))
             inputs = np.array(inputs)
-            inference = self._launcher.launch(inputs)
+
+            if self._launcher.inferenced_check(batch):
```

Comment:
Could you explain more about `inferenced_check` function? It seems like always return `False`, when `if` statement(L53-L56) will be used?

Reply:
I defined `inferenced_check` in `datumaro/plugins/explorer.py`. This function is related to explorer to check whether the item is inferenced before or not.